src/components/Mobile/MobileProject/MobileProject.js

```diff
@@ -48,22 +49,23 @@ const MobileProject = () => {
             </TabPanel>
             <MobileProjectBar />
             <div className="react-tabs__tab-container mobile-nav">
-              <span
-                className="react-tabs__tab-inner mobile-nav__menu"
-                onClick={openSidebar}
-              >
-                <MenuIcon />
-              </span>
+              {forWebComponent && (
```

Comment:
needs to be `! forWebComponent` because the menu icons disappeared now 😅

Reply:
Oh no 😱